### ISSUE TITLE
Keyboard module is not listening keyboardWillShow on android

### ISSUE THREAD

**Issue by agrcrobles** [facebook/react-native#15387]
### Is this a bug report?
yes

### Environment

1. `react-native -v`: latest
2. `platform`: android

### Steps to Reproduce

The following code on `componentWillMount` works on ios but doesn't work in android

```
this.keyboardWillShowListener = Keyboard.addListener(
  "keyboardWillShow",
  () => console.log("do something")
);
```

### Expected Behavior
`keyboardWillShow` native event is triggered on android

### Actual Behavior
It doesn't trigger so I had to use `keyboardDidShow` event instead.

### Reproducible Demo

https://snack.expo.io/r1eRxoNvb

**Comment by agrcrobles** [comment]
upps,
Reading at the docs again I found that `keyboardWillShow` is not available on Android since there is no native corresponding event.

https://facebook.github.io/react-native/docs/keyboard.html

**Comment by agrcrobles** [comment]
Closed #15387
